### PR TITLE
Add actions for stale-in-review, stale, size, and area labels

### PR DESCRIPTION
**Pull request by ggriffiths**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+area/build:
```
Comment (piyush-nimbalkar) [inline .github/labeler.yml:1]: Can you add the following
```
area/specs:
- deploy/**/*

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- build/**/*
+area/ci:
+- .travis
+- .github/**/*
+area/components:
+- drivers/storage/portworx/component/**/*
```
Comment (piyush-nimbalkar) [inline .github/labeler.yml:7]: In future we could create area for different components like CSI, Security, etc. I think it's okay for now.

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+area/manifest:
+- drivers/storage/portworx/manifest/**/*
+area/portworx:
+- drivers/storage/portworx/*
+area/controller:
+- pkg/controller/**/*
+area/test:
```
Comment (piyush-nimbalkar) [inline .github/labeler.yml:14]: Let's call this `area/integration-test`

**Review by piyush-nimbalkar — APPROVED** [review]
Thanks for doing this, Grant!

File: .github/labeler.yml (hunk 1 above)
Comment (ggriffiths) [reply .github/labeler.yml:3]: added

**Pull request merged**